grails-app/views/inventory/_filters.gsp

```diff
@@ -11,24 +11,39 @@
 							 class="text medium" style="width:100%;"/>
 			</div>
 			<div class="filter-list-item">
-				<g:selectCategory id="category"
-								  name="categoryId"
-								  class="chzn-select-deselect"
-								  noSelection="['':'']"
-								  data-placeholder="Select a category"
-								  value="${commandInstance?.category?.id}"/>
+				<g:selectCategoryViaAjax id="category"
+										 name="categoryId"
+										 class="ajaxSelect2"
+										 noSelection="['':'']"
+										 data-placeholder="Select a category"
+										 value="${commandInstance?.category?.id}"
+										 data-allow-clear="true"
+										 data-ajax--url="${request.contextPath }/json/findCategory"
+										 data-ajax--cache="true"/>
 			</div>
 			<div class="filter-list-item">
-				<g:selectTags name="tags" noSelection="['':'']"
-							  value="${commandInstance.tags*.id}"
-							  data-placeholder="Select a tag"
-							  class="chzn-select-deselect"/>
+				<g:selectTagsViaAjax id="tags"
+										 name="tags"
+										 class="ajaxSelect2"
+										 noSelection="['':'']"
+										 multiple="true"
+					                     data-placeholder="Select a tag"
```

Comment:
I don't know why the indents look so weird here, locally they are one under another